build/configs/artik053/audio/defconfig

```diff
@@ -421,11 +421,6 @@ CONFIG_AUDIO_DEVICES=y
 # CONFIG_AUDIO_NULL is not set
 CONFIG_AUDIO_ALC5658=y
 CONFIG_ALC5658_INITVOLUME=250
-CONFIG_ALC5658_INFLIGHT=2
-CONFIG_ALC5658_MSG_PRIO=1
-CONFIG_ALC5658_BUFFER_SIZE=2048
```

Comment:
CONFIG_ALC5658_BUFFER_SIZE & NUM_BUFFERS using when CONFIG_AUDIO_DRIVER_SPECIFIC_BUFFERS enabled.
see line962 in alc5658.c

Reply:
Yes, but now, in this defconfig CONFIG_AUDIO_DRIVER_SPECIFIC_BUFFER is NOT enabled.

Reply:
Agreed that in latest code, these are used. Hence removed the dependency.